status-messages/cloudformation.yml

```
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  HyP3URL:
+    Type: String
+
+  QueueName:
+    Type: String
+
+  MattermostPAT:
+    Type: String
+
+Resources:
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${Lambda}"
```

I'm not seeing a default retention period specified in the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html#cfn-logs-loggroup-retentionindays), so I'm not sure how long logs would be saved if we didn't specify a value. 30 days seems reasonable to me.
```suggestion
      LogGroupName: !Sub "/aws/lambda/${Lambda}"
      RetentionInDays: 30
```